patternfly/patternfly-elements
# CPFED-3940 primary detail

## Primary Details Component
Primary details (a name I do not understand, but PF is using it)
This component is needed for Ansible.com and the Customer Portal to update their navigation. It mimics "Primary Detail simple list in card" from: https://www.patternfly.org/v4/demos/primary-detail


### Related issue

- CPFED-3940: Make the Primary Detail component


### Preview


Link(s) to demo page(s) where this element can be viewed:
- [Link](https://5e6089f7c8e38b0008963801--happy-galileo-ea79c4.netlify.com/examples/) 

### Testing instructions



1. Go to the demo page `/elements/pfe-primary-detail/demo/`
2. Click/press/tab to & press Enter/Space on the different navigation options on the left side, make sure the corresponding content shows up.
3. Want to make sure the aria attributes are appropriate (cc: @kelsS ?)


#### Browser requirements

Your component should work in all of the following environments:



### Ready-for-merge Checklist

Check off items as they are completed.  Feel free to delete items if they are not applicable.



### Merging

Please **squash** when merging and ensure your commit message uses [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/#summary) formatting.

**Be sure to share your updates with the [patternfly-elements-contribute@redhat.com](mailto:patternfly-elements-contribute@redhat.com) mailing list!**
